### DISCUSSION TITLE
QoL主城快捷tp（配合抠图icon）

### DISCUSSION THREAD

**Discussion by yimo0908** [DeepseaXX/aakaigua#3] [Show and tell]
![1](https://github.com/user-attachments/assets/c49a2d91-4145-453e-90f5-ec1f32991ef6)
![2](https://github.com/user-attachments/assets/f2a40352-5014-4f66-8dc3-b699adb46d56)
![3](https://github.com/user-attachments/assets/3bcecd67-bc51-43e2-97ff-0385f9f76970)
![4](https://github.com/user-attachments/assets/0e378315-1df2-4be6-b43c-2e1f05731b64)

```
H4sIAAAAAAAACqVV308TQRD+V8iej2c5yrW93KMmPplo1MQHwwvYBxJjGqkkhvRBqlAoqSAowh2UWqAFQr1aoddSyj9zs9f9L5xrUjtHNDHdXO7HfrP7fTM7s3ML7E76XSrJTJZkKpuOMnPhD4Ijlb3GD8+9gmLh2WMczj1k5ovhlLnBFNOMa3ospigP7iMyg8h4OjUmslXxrchzN1DPIzz7CBdrEU3FewqHT5gZjNjM0xQaDFWfyqj/4E4YhqLw0iK3q1C7JBLwse61z+QlEvGEllAUsbzhfy/SEPoAd12vtQ52Wy4Kw5jAKC6aIntNQ8idQGWpt16VJY8iebkWJucHHbG81ut2wel47V1ZiUmU+LkTlvBaa+BsBRKbOVl+HVNgnd7i76x6NzVYPYBuU45fj6H//pYD1rnIlogExaT8xxMgrj5g0XD3B60igo3Mfxfz6x/t83zD3/5CnSeY7ObA4a5fsER5H2yHlmgYltqiOJaQ84vvXN4qVIKNvkVBfbout8/E4jElJ5iU89gi+HYFz6qwmlBboRJhePQQghaRX+H2uX9NszxGQakYsJOC1e0V6uAcIilNcxgePQY8xl5rDz41e5UyLDX41xYvBb1h2DL+YpStXb5xwvdsv92gQm7Ff38hPgct/T/4EX6JqMrm+8/p58zUdZUF9uHCV8xMv3mbxJ8hMyciMXwP1qND9wZG9EDL9IlYNDKJl8YyvwG5dRyabgcAAA==
```

**Comment by DeepseaXX** [comment]
已整合至[aakaigua/_posts/2024-09-30-qolbar_share.md at master · DeepseaXX/aakaigua](https://github.com/DeepseaXX/aakaigua/blob/master/_posts/2024-09-30-qolbar_share.md)
